### PR TITLE
Feat/839 login submit

### LINKED ISSUE #839: Login form doesn't submit when enter key pressed
Tested on Windows 10 - Chrome & Firefox.
Create account page form does submit on enter key press though.

### PR DESCRIPTION
**Pull request by jamesxu123**
### Tickets:

- HCK-#839 

### List of changes:

- Switched Button in Login form to SubmitBtn
- This fixes the submit on enter problem and should make components more consistent

### Type of change:

Please delete options that aren't relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### PR Checklist:

- [x] Merged `develop` branch (before testing)
- [x] Linted my code locally
- [ ] Listed change(s) in the Changelog
- [x] Tested all links in project relevant browsers
- [x] Tested all links on different screen sizes
- [x] Referenced all useful info (issues, tasks, etc)

### Screenshots:
Shouldn't have any visual changes:
image

resolves #839

### CODE REVIEW

File: src/pages/Login/index.tsx [hunk 1]
```diff
@@ -97,11 +98,7 @@ const LoginPage: React.FC = () => {
             hasResetLink={true}
             placeholder="your_password"
           />
-          <Box pr={'5px'} pt={'36px'} pb={'40px'}>
```
Comment (logan-r) [inline src/pages/Login/index.tsx:100]: Does removing this change the padding on SubmitBtn?

Comment (logan-r) [reply src/pages/Login/index.tsx:100]: Nvm, tested locally, there are no changes to padding :)

**Review by logan-r — APPROVED** [review]
This break the create account page:

[Pressed enter too soon, ignore this - reviewing PR rn will update asap]

**Comment by logan-r** [comment]
We should generalize this to other forms. Create/edit account, application, etc. doesn't handle enter as onSubmit. @jamesxu123 can you look into https://github.com/hackmcgill/dashboard/issues/895 when you get a chance?

**Comment by jamesxu123** [comment]
Yeah sure! I'll take a look.

**Pull request merged**